h/buildext.py

```diff
@@ -274,6 +275,11 @@ def check_sentry_dsn_is_public(dsn):
 
 
 parser = argparse.ArgumentParser('hypothesis-buildext')
+parser.add_argument('--bouncer',
```

Comment:
For consistency please do what I originally asked, which was to use `dest='bouncer_url'` so that it's referenced as `bouncer_url` internally. It's probably also worth setting `metavar='URL'`.

This should also be moved down with the other parameters which represent service URLs. I know you're only trying to keep things alphabetised as I always moan about that, but this is different, as it's the order in which people see the arguments. Generic args like `--debug` should come first, I think.